packages/connect/src/api/getDeviceState.ts

```diff
@@ -1,15 +1,19 @@
 // origin: https://github.com/trezor/connect/blob/develop/src/js/core/methods/GetDeviceState.js
 
 import { AbstractMethod } from '../core/AbstractMethod';
+import { ERRORS } from '../constants';
 
 export default class GetDeviceState extends AbstractMethod<'getDeviceState'> {
     init() {
         this.requiredPermissions = [];
     }
 
     run() {
-        return Promise.resolve({
-            state: this.device.getExternalState(),
-        });
+        const state = this.device.getState();
+        if (!state?.staticSessionId) {
+            throw ERRORS.TypedError('Runtime', 'Device state not set');
+        }
```

Comment:
there is change here, right? previously it would return undefined?
```javascript
 device.getExternalState() {
        return this.externalState[this.instance];
```

Reply:
exactly, but it shouldn't